### DISCUSSION TITLE
Do zero padding

### DISCUSSION THREAD

**Discussion by KES777** [dbeaver/dbeaver#14460] [General]
This is hard to read when data is not adjusted

![image](https://user-images.githubusercontent.com/799021/99384721-81c2b400-28d8-11eb-8c23-dbad8f7d794b.png)

**Describe the solution you'd like**
Display zeroes at the end by default

**Comment by uslss** [comment]
thanks for suggestion

**Comment by uslss** [comment]
![image](https://user-images.githubusercontent.com/31996417/139810459-0b56ab81-c817-4ed0-8e26-a69d2279f69b.png)
Right-justify date/time values could be unchecked in preferences

**Reply by KES777** [reply]
Probably this is my mistake when I opening the issue, but

Notice: Justifying is not padding

Actually I am asking for padding:

![image](https://user-images.githubusercontent.com/799021/140172189-b36fa6c7-7de0-49f1-8b4b-2f4f24442c93.png)
